### PR TITLE
[pascal/es-es] Translate Pascal to es_ES 

### PR DESCRIPTION
**Pull request by AlburIvan**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by divayprakash** [comment]
@Menelion Please review this further for the translation.

File: es-es/pascal-es.html.markdown [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+           // puede especificar la longitud de la cadena para minimizar el uso de memoria
+    my_str: ch_array;
+           // Puedes declarar variables de tipos personalizados.
+    my_2d : md_array;
+           // Las matrices de tamaño dinámico deben dimensionarse antes de que puedan usarse.
+
+    // tipos de datos enteros adicionales
+    b    : byte;     // rango [0..255]
+    shi  : shortint; // rango [-128..127]
+    smi  : smallint; // rango [-32,768..32,767] (entero estandar)
```
Comment (Menelion) [inline es-es/pascal-es.html.markdown]: estándar

File: es-es/pascal-es.html.markdown [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    write('Escriba un entero: ');
+    readln(i);
+    dummy := 4; // dummy no cambiará el valor fuera del procedimiento
+End;
+
+Begin // bloque de programa principal
+    dummy := 3;
+    get_integer(i, dummy);
+    writeln(i, '! = ', factorial_recursion(i));
+    // outputs i!
```
Comment (Menelion) [inline es-es/pascal-es.html.markdown]: ¿Hay que traducirlo?

File: es-es/pascal-es.html.markdown [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+var a,b : integer;
+```
+
+```pascal
+program Learn_More;
+//Aprendamos sobre los tipos de datos y sus operaciones.
+
+const
+    PI = 3.141592654;
+    GNU = 'GNU's Not Unix';
```
[the remark this answers is not in the document]
Comment (AlburIvan) [reply es-es/pascal-es.html.markdown]: Arreglado. No obstante los tres apostrofes también existen en el archivo original de ingles

**Comment by AlburIvan** [comment]
should be done now :) @Menelion

**Comment by Menelion** [comment]
¡Muchas gracias Ivan! :)

**Pull request merged**